api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadServiceDatabase.java

```diff
@@ -190,4 +215,43 @@ public VehicleReadServiceDatabase(VehicleReadDao vehicleReadDao) {
             return null;
         }
     }
+
+    protected DvlaVehicle mapDvlaVehicleSqltoJson(DvlaVehicle vehicle) {
+
+        if (vehicle != null) {
+            DvlaVehicle jsonVehicle = new DvlaVehicle();
+
+            jsonVehicle.setId(vehicle.getId());
+            jsonVehicle.setRegistration(vehicle.getRegistration());
+
+            jsonVehicle.setDvlaVehicleId(vehicle.getDvlaVehicleId());
+
+            jsonVehicle.setFirstRegistrationDate(vehicle.getFirstRegistrationDate());
+            jsonVehicle.setManufactureDate(vehicle.getManufactureDate());
+            jsonVehicle.setColour1(vehicle.getColour1());
+            if (vehicle.getColour2() != null) {
```

Comment:
I think we don't need to check if it's null. If it is then `jsonVehicle.setColour2(null)` wouldn't break or change anything. :?